docs/recipes/install/centos7/x86_64/warewulf/pbspro/steps.tex

```diff
@@ -266,7 +266,7 @@ \subsection{3rd Party Libraries and Tools} \label{sec:3rdparty}
 \vspace*{0.8cm}
 \subsection{Optional Development Tool Builds} \label{sec:3rdparty_intel}
 \vspace*{0.2cm}
-\input{common/pxse_enabled_builds}
+\input{common/psxe_enabled_builds}
```

Comment:
Maybe I'm missing it, but wouldn't the underlying latex files have to change their names too?

`pxse_enabled_builds.tex -> psxe_enabled_builds.tex`

Reply:
Oops - I did miss it. The file rename is indeed in the PR.